Elemental/Components/AeCard.razor

```diff
@@ -0,0 +1,36 @@
+﻿@*
+  Copyright (c) 2020 Apption Corporation
+
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*@
+
+@inherits HtmlElement
+
+  <div class="@_classname">
```

Comment:
Should probably [splat any unmapped attributes](https://docs.microsoft.com/en-us/aspnet/core/blazor/components/?view=aspnetcore-3.1#attribute-splatting-and-arbitrary-parameters) onto this outer div (except for the `class` one, there should be an inherited element in `HtmlElement` called `InputAttributesWithoutClass` you can splat